cricPredict/User_profile/urls.py

```diff
@@ -1,12 +1,10 @@
 from django.conf.urls import url
-from django.contrib import admin
-from django.urls import path, include
-from .views import UserAPIView, Login, Logout
-from . import views
+from .views import UserAPIView, Login, Logout, Signup
 
 urlpatterns = [
-    url(r'^user/$', UserAPIView.as_view(), name="User"),
-    url(r'^user/(?P<pk>[\d]+)/$', UserAPIView.as_view(), name="League"),
-    url(r'^$', Login.as_view(), name="Login"),
+    url(r'^list/$', UserAPIView.as_view(), name="User"),
+    url(r'^signup/$', Signup.as_view(), name='Signup'),
+    url(r'^methods/(?P<pk>[\d]+)/$', UserAPIView.as_view(), name="User APi"),
```

Comment:
why is this URL named methods? a URL should always represent some business object/use case

Reply:
This api has get, put, patch, delete methods. I couldn't think of some name which covers all these.